Rakefile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
I imagine this is automatically added by gem. It's really necessary? I would delete these comments in all files :sweat_smile: